svc/FetchService.ts

```diff
@@ -78,7 +78,7 @@ export class FetchService extends HoistService {
                 },
                 aborter
             );
-            return this.NO_JSON_RESPONSES.includes(r.status) ? null : r.json();
+            return this.NO_JSON_RESPONSES.includes(r.status) ? null : this.jsonAsync(r);
```

Comment:
wonder if we should return `undefined` on line 81  or `null` on line 293-- seems inconsistent as it is